boards/t420/t420.config

```diff
@@ -27,7 +27,7 @@ export CONFIG_TPM=y
 export CONFIG_BOOTSCRIPT=/bin/gui-init
 export CONFIG_BOOT_REQ_HASH=n
 export CONFIG_BOOT_REQ_ROLLBACK=n
-export CONFIG_BOOT_KERNEL_ADD=""
+export CONFIG_BOOT_KERNEL_ADD="intel_iommu=on"
```

Comment:
why not in the first place, when you add the file?

Reply:
I don't quite understand what you mean?